dbt_sugar/core/yaml_module/yaml_helpers.py

```diff
@@ -25,3 +25,14 @@ def open_yaml(path: Path) -> Dict[str, Any]:
                 return yaml_dict
             raise YAMLFileEmptyError(f"The following file {path.resolve()} seems empty.")
     raise FileNotFoundError(f"File {path.resolve()} was not found.")
+
+
+def save_yaml(path: Path, data: Dict[str, Any]) -> None:
```

Comment:
When I merge my PR (the one where you pointed out potential namespace collision with the module `yaml` you might want to make sure you stick that function in the new place since it totally belongs there.

Reply:
will rebase!

Reply:
You can go ahead the PR you approved containing the yaml stuff is now merged.